hook.py

```diff
@@ -8,10 +8,14 @@
 
 
 async def enable(services):
-    Popen(['haproxy', '-q', '-f', 'plugins/ssl/templates/haproxy.conf'])
+    haproxy_conf = Path('plugins/ssl/templates/haproxy.conf')
+    user_conf = Path('plugins/ssl/conf/haproxy.conf')
+    if user_conf.is_file():
+        haproxy_conf = user_conf
+    Popen(['haproxy', '-q', '-f', haproxy_conf])
     logging.debug('Serving at https://127.0.0.1:8443')
     cert = Path("plugins/ssl/conf/insecure_certificate.pem")
```

Comment:
instead of checking on the existence of `insecure_certificate.pem` I think we should make this check a bit more robust. Off the top of my head a couple ways come to mind: 

 1. read the config file, something like: 

```python
if "insecure_certificate.pem" in haproxy_conf.read_text():
    logging.warn('Found insecure SSL...')
```

That should be a little bit more robust I think in case someone leaves the file behind.


2. The more surefire way way to do the check would be to make a request to the newly started haproxy process and compare its cert with the insecure one.